lib/mix/tasks/talon.gen.theme.ex

```diff
@@ -220,6 +223,41 @@ defmodule Mix.Tasks.Talon.Gen.Theme do
   end
   defp gen_generators(config), do: config
 
+  def gen_dashboard_template(%{dashboard: true} = config) do
+    binding = Kernel.binding() ++
+      [title_txt: gettext("Dashboard"),
```

Comment:
this should be coded at `[title_txt: ~s/dgettext("Talon", "Dashboard")/]`. This way the generated code contains a call to dgettext("Talon", "Dashboard") which is compiled in the context of the project, and therefore uses the projects Gettext module.